pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java

```diff
@@ -382,7 +382,7 @@ void run() throws PulsarAdminException {
         @Parameter(description = "tenant/namespace", required = true)
         private java.util.List<String> params;
 
-        @Parameter(names = { "--maxSubscriptionsPerTopic", "-m" }, description = "Max subscriptions per topic",
+        @Parameter(names = { "--max-subscriptions-per-topic", "-m" }, description = "Max subscriptions per topic",
```

Comment:
Could you please deprecate the old parameter name instead of deleting it directly? We can make the old one does not appear in the CLI help but also able to use it. Then we can try to delete it in the 2.8.0 or later.

Reply:
topic-level `get-max-subscriptions-per-topic` was added on 2020-12-16
namespace-level `maxSubscriptionsPerTopic` was added on 2020-12-12

It seems that it has not been released yet, can it be modified directly?

Reply:
@codelipenghui

Reply:
Ok